examples/mean/tests/Makefile

```diff
@@ -1,33 +1,21 @@
-
 TOPLEVEL_LANG ?= verilog
 
-ifneq ($(TOPLEVEL_LANG),verilog)
-
-all:
-	@echo "Skipping example due to TOPLEVEL_LANG=$(TOPLEVEL_LANG) not being verilog"
-clean::
-
-else
-
-TOPLEVEL := mean_sv
-
 PWD=$(shell pwd)
 
-VHDL_SOURCES = $(PWD)/../hdl/mean_pkg.vhd
-VHDL_SOURCES += $(PWD)/../hdl/mean.vhd
-
-ifeq ($(SIM),questa)
-COMPILE_ARGS = -mixedsvvh
+ifeq ($(TOPLEVEL_LANG),verilog)
+    VERILOG_SOURCES = $(PWD)/../hdl/mean.sv
+else ifeq ($(TOPLEVEL_LANG),vhdl)
+    VHDL_SOURCES = $(PWD)/../hdl/mean_pkg.vhd
```

Comment:
```suggestion
    VHDL_SOURCES += $(PWD)/../hdl/mean_pkg.vhd
```

Reply:
Is this really necessary? Tried to change these in all the makefiles, but then the file changed too much for git to detect the file moving from /test to /test_mixedlang.

Reply:
No, certainly not necessary.